### PR TITLE
Updates "Private Key" section from README.md

### PR DESCRIPTION
**Pull request by zedtux**
This PR adds a convenient rake task to generate the private key and update accordingly the `README.md` file.

### CODE REVIEW

**Comment by zedtux** [comment]
Actually, my first commit was wrong since it was exporting a public key.

I have changed it in order to copy the private key, but now it doesn't work since I get the error:
```

OpenSSL::PKey::RSAError at /jwks.json

Neither PUB key nor PRIV key: no start line
```

I have checked the key file with:

```
ssh-keygen -y -f ./lib/oidc_provider_key.pem
```

And it works fine (it ask me the passphrase and then print the public key). So I'm a bit lost on this one.

File: lib/tasks/oidc_provider.rake [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# require_relative '../../app/models/oidc_provider/id_token'
```
Comment (willtcarey) [inline lib/tasks/oidc_provider.rake]:
```suggestion
```

**Review by willtcarey — COMMENTED** [review]
Just one small comment change and then I think we can merge this. Thanks for contributing!

**Comment by willtcarey** [comment]
Thanks!

**Pull request merged**